README.md

```diff
@@ -1,8 +1,76 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Nihar Sheth . Jarvis Consulting
 
+I am a computer engineering graduate from Ryerson University. I chose to pursue this field of study early on as I have always had an interest in computers and working with them. During my undergrad I gained experience and exposure to various fields such as software design, computer networks and security and data engineering. I found that data engineering captured my interest and I decided to further my career in it. What fascinates me about data engineering is its versatility as it applies to nearly every industry out there, and its importance is only growing with time.
```

Comment:
I am a computer engineering graduate from Ryerson University. I chose to pursue this field of study early on as I always have been interested in computers and working with them. During my undergrad, I gained experience and exposure to various fields such as software design, computer networks and security and data engineering. I found that software and data engineering captured my interest, and I decided to further my career. What fascinates me about data engineering is its versatility as it applies to nearly every industry out there, and its importance is only growing with time.
		
I did some minor fix for you.